static/app/components/performanceOnboarding/sidebar.spec.tsx

```diff
@@ -176,16 +183,21 @@ describe('Sidebar > Performance Onboarding Checklist', function () {
     );
   });
 
-  it('checklist feature enabled > navigate to performance page > project without performance support', async function () {
-    ProjectsStore.loadInitialData([
-      ProjectFixture({platform: 'elixir', firstTransactionEvent: false}),
-    ]);
+  fit('checklist feature enabled > navigate to performance page > project without performance support', async function () {
+    console.log('organization projects before', organization.projects);
+    const project = ProjectFixture({
+      platform: 'nintendo-switch',
+      firstTransactionEvent: false,
+    }) as Project & {platform: PlatformKey};
+    ProjectsStore.loadInitialData([project]);
+    console.log('organization projects after', organization.projects);
```

Comment:
This is the test I have problem with. Both console logs return the same project which is not what I'm adding here what am I missing?
```
organization projects before [
        {
          id: '2',
          slug: 'project-slug',
          name: 'Project Name',
          access: [ 'project:read' ],
          allowedDomains: [ '*' ],
          hasAccess: true,
          isMember: true,
          isBookmarked: false,
          team: {
            id: '1',
            slug: 'team-slug',
            name: 'Team Name',
            access: [Array],
            teamRole: null,
            isMember: true,
            memberCount: 0,
            avatar: [Object],
            flags: [Object],
            externalTeams: [],
            projects: [],
            hasAccess: false,
            isPending: false
          },
          teams: [],
          environments: [],
          features: [],
          eventProcessing: { symbolicationDegraded: false },
          dateCreated: '2017-10-17T02:41:20.000Z',
          digestsMaxDelay: 0,
          digestsMinDelay: 0,
          dynamicSamplingBiases: null,
          firstEvent: null,
          firstTransactionEvent: false,
          groupingAutoUpdate: false,
          groupingConfig: '',
          hasCustomMetrics: false,
          hasFeedbacks: false,
          hasNewFeedbacks: false,
          hasMinifiedStackTrace: false,
          hasProfiles: false,
          hasReplays: false,
          hasSessions: false,
          hasMonitors: false,
          isInternal: false,
          organization: {
            id: '3',
            slug: 'org-slug',
            name: 'Organization Name',
            links: [Object],
            access: [Array],
            status: [Object],
            experiments: {},
            scrapeJavaScript: true,
            features: [],
            onboardingTasks: [],
            aiSuggestedSolution: false,
            alertsMemberWrite: false,
            allowJoinRequests: false,
            allowSharedIssues: false,
            attachmentsRole: '',
            availableRoles: [],
            avatar: [Object],
            codecovAccess: false,
            dataScrubber: false,
            dataScrubberDefaults: false,
            dateCreated: '2017-10-17T02:41:20.000Z',
            debugFilesRole: '',
            defaultRole: '',
            enhancedPrivacy: false,
            eventsMemberAdmin: false,
            githubOpenPRBot: false,
            githubPRBot: false,
            githubNudgeInvite: false,
            isDefault: false,
            isDynamicallySampled: true,
            isEarlyAdopter: false,
            genAIConsent: false,
            openMembership: false,
            pendingAccessRequests: 0,
            quota: [Object],
            relayPiiConfig: null,
            require2FA: false,
            safeFields: [],
            scrubIPAddresses: false,
            sensitiveFields: [],
            aggregatedDataConsent: false,
            storeCrashReports: 0,
            trustedRelays: [],
            teams: [],
            projects: [],
            orgRoleList: [Array],
            teamRoleList: [Array]
          },
          plugins: [],
          processingIssues: 0,
          relayPiiConfig: '',
          resolveAge: 0,
          safeFields: [],
          scrapeJavaScript: true,
          scrubIPAddresses: false,
          sensitiveFields: [],
          subjectTemplate: '',
          verifySSL: false
        }
      ]

      at Object.log (static/app/components/performanceOnboarding/sidebar.spec.tsx:187:13)

    console.log
      organization projects after [
        {
          id: '2',
          slug: 'project-slug',
          name: 'Project Name',
          access: [ 'project:read' ],
          allowedDomains: [ '*' ],
          hasAccess: true,
          isMember: true,
          isBookmarked: false,
          team: {
            id: '1',
            slug: 'team-slug',
            name: 'Team Name',
            access: [Array],
            teamRole: null,
            isMember: true,
            memberCount: 0,
            avatar: [Object],
            flags: [Object],
            externalTeams: [],
            projects: [],
            hasAccess: false,
            isPending: false
          },
          teams: [],
          environments: [],
          features: [],
          eventProcessing: { symbolicationDegraded: false },
          dateCreated: '2017-10-17T02:41:20.000Z',
          digestsMaxDelay: 0,
          digestsMinDelay: 0,
          dynamicSamplingBiases: null,
          firstEvent: null,
          firstTransactionEvent: false,
          groupingAutoUpdate: false,
          groupingConfig: '',
          hasCustomMetrics: false,
          hasFeedbacks: false,
          hasNewFeedbacks: false,
          hasMinifiedStackTrace: false,
          hasProfiles: false,
          hasReplays: false,
          hasSessions: false,
          hasMonitors: false,
          isInternal: false,
          organization: {
            id: '3',
            slug: 'org-slug',
            name: 'Organization Name',
            links: [Object],
            access: [Array],
            status: [Object],
            experiments: {},
            scrapeJavaScript: true,
            features: [],
            onboardingTasks: [],
            aiSuggestedSolution: false,
            alertsMemberWrite: false,
            allowJoinRequests: false,
            allowSharedIssues: false,
            attachmentsRole: '',
            availableRoles: [],
            avatar: [Object],
            codecovAccess: false,
            dataScrubber: false,
            dataScrubberDefaults: false,
            dateCreated: '2017-10-17T02:41:20.000Z',
            debugFilesRole: '',
            defaultRole: '',
            enhancedPrivacy: false,
            eventsMemberAdmin: false,
            githubOpenPRBot: false,
            githubPRBot: false,
            githubNudgeInvite: false,
            isDefault: false,
            isDynamicallySampled: true,
            isEarlyAdopter: false,
            genAIConsent: false,
            openMembership: false,
            pendingAccessRequests: 0,
            quota: [Object],
            relayPiiConfig: null,
            require2FA: false,
            safeFields: [],
            scrubIPAddresses: false,
            sensitiveFields: [],
            aggregatedDataConsent: false,
            storeCrashReports: 0,
            trustedRelays: [],
            teams: [],
            projects: [],
            orgRoleList: [Array],
            teamRoleList: [Array]
          },
          plugins: [],
          processingIssues: 0,
          relayPiiConfig: '',
          resolveAge: 0,
          safeFields: [],
          scrapeJavaScript: true,
          scrubIPAddresses: false,
          sensitiveFields: [],
          subjectTemplate: '',
          verifySSL: false
        }
      ]
     ```